bin/index.js

```diff
@@ -204,6 +272,20 @@ const promptMainMenu = (manifest) => {
     });
   }
 
+  if (!manifest.exchangeUrl) {
+    choices.splice(choices.length - 2, 0, {
+      name: 'Add an Exchange URL',
+      value: addExchangeUrl
+    });
+  }
+
+  if (!manifest.iconPath) {
+    choices.splice(choices.length - 2, 0, {
+      name: 'Add an icon path',
+      value: addIconPath
+    });
+  }
+
```

Comment:
Splicing at `choices.length - 2` seems brittle. I'd prefer we rearrange this to build the choices in order. So create an empty array, add the extension configuration view option if needed, then add the event, condition, action, and shared module options, then add the exchange URL and icon path options if needed, then the separator, then the I'm done option.